ui/src/constants.ts

```diff
@@ -3,5 +3,12 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { QuickInputButton, ThemeIcon } from "vscode";
+import { localize } from "./localize";
+
 export const resourcesProvider: string = 'Microsoft.Resources';
 export const storageProvider: string = 'Microsoft.Storage';
+
+export namespace AzExtQuickInputButtons {
+    export const LearnMore: QuickInputButton = { iconPath: new ThemeIcon('question'), tooltip: localize('clickHere', 'Click here to learn more about this step.') }
```

Comment:
I don't like "Click here" because I feel like people know how buttons work. I would suggest we just go with `Learn more` which matches [this](https://github.com/microsoft/vscode-azuretools/blob/main/ui/src/DialogResponses.ts#L14)

Also, I anticipate people will want to customize this text anyways, so we might want `learnMoreTooltip` added as an option next to `learnMoreLink`

Reply:
> Also, I anticipate people will want to customize this text anyways, so we might want learnMoreTooltip added as an option next to learnMoreLink

So I originally had an optional tooltip property to customize it, but it makes it impossible to have a constant LearnMore button.  That's not really a problem per se, but does make the button handling check a little less reliable/clean.  

I also thought about it, and I'm not really sure how much value there is in having the tooltip be customizable.  At most, I feel like you could just say "Learn more about _____" which isn't that useful to me.